Flaky unit test

I got a flaky test during setting up the dev env

```
        # Should have 3 finished iteration entries
>       assert len(pr.history) == 6
E       AssertionError: assert 5 == 6
E        +  where 5 = len([{'iteration': {'avg_time_per_item': 2.1457672119140625e-06, 'current_item': 1, 'desc': 'level3', 'duration': 2.145767...tem': 8.821487426757812e-06, 'current_item': 1, 'desc': 'level1', 'duration': 8.821487426757812e-06, ...}, 'level': 1}])
E        +    where [{'iteration': {'avg_time_per_item': 2.1457672119140625e-06, 'current_item': 1, 'desc': 'level3', 'duration': 2.145767...tem': 8.821487426757812e-06, 'current_item': 1, 'desc': 'level1', 'duration': 8.821487426757812e-06, ...}, 'level': 1}] = <planqtn.progress_reporter_test.ConcatenatingProgressReporter object at 0x15814dfa0>.history

planqtn/progress_reporter_test.py:139: AssertionError
```

--- a/planqtn/progress_reporter.py
+++ b/planqtn/progress_reporter.py
@@ -254,7 +254,7 @@ class ProgressReporter(abc.ABC):
         for item in iterable:
             yield item
             bottom_iterator_state.update()
-            if time.time() - time_last_report > self.iteration_report_frequency:
+            if time.time() - time_last_report >= self.iteration_report_frequency:
                 time_last_report = time.time()
 
                 self.log_result(

